gems/pending/util/miq-process.rb

```diff
@@ -205,6 +205,13 @@ def self.find_pids(cmd)
           pids << pid.to_i
         end
       end
+    when :macosx
+      require 'sys/proctable'
```

Comment:
Is this Mac specific?  We check /proc on linux as well, and if this gem encapsulates the whole thing we can remove the custom code up above.